build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
         opensearch_version = System.getProperty("opensearch.version", "2.10.0-SNAPSHOT")
         isSnapshot = "true" == System.getProperty("build.snapshot", "true")
         buildVersionQualifier = System.getProperty("build.version_qualifier", "")
-        kotlin_version = System.getProperty("kotlin.version", "1.6.10")
+        kotlin_version = System.getProperty("kotlin.version", "1.8.21")
```

Comment:
@bowenlan-amzn  I am not sure we need upgrade this in 2.x branch, can you confirm

Reply:
Yes, this should be fine